.circleci/config.yml

```diff
@@ -189,29 +189,37 @@ workflows:
             - unit_testing_php_74
             - unit_testing_php_80
             - js
-      #- e2e_chrome_wp_previous_major:
```

Comment:
Did you fix the issues with WP 5.7 ?
I disabled this step while there were tests that were problematic.

Reply:
@olafleur No, I didn't. I must have accidentally re-enabled that. I'll disable it again.

Reply:
See for example https://app.circleci.com/pipelines/github/godaddy-wordpress/coblocks/7700/workflows/3aa3739e-eaf5-4c44-a4b5-c52ad4d45c7a/jobs/35347
Which is the current status of the 5.7 tests